homeassistant/components/airly/__init__.py

```diff
@@ -61,10 +58,13 @@ def update_interval(hass, unload=False):
         )
 
     websession = async_get_clientsession(hass)
-    upd_interval = update_interval(hass)
+    # Change update_interval for other Airly instances
+    update_interval = set_update_interval(
+        hass, len(hass.config_entries.async_entries(DOMAIN))
+    )
 
     coordinator = AirlyDataUpdateCoordinator(
-        hass, websession, api_key, latitude, longitude, upd_interval
+        hass, websession, api_key, latitude, longitude, update_interval
     )
     await coordinator.async_refresh()
```

Comment:
Let's postpone setting up Airly if we can't currently connect.

```suggestion
    await coordinator.async_refresh()
    
    if not light_coordinator.last_update_success:
        raise ConfigEntryNotReady
```

Reply:
Ehh, I knew I had forgotten something ;)
Fixed.